adapters/handlers/rest/handlers_aliases.go

```diff
@@ -75,11 +75,21 @@ func (s *aliasesHandlers) getAlias(params schema.AliasesGetAliasParams,
 				WithPayload(errPayloadFromSingleErr(err))
 		}
 	}
-
-	aliasesResponse := &models.AliasResponse{Aliases: aliases}
+	if len(aliases) == 0 {
+		return schema.NewAliasesGetAliasNotFound()
+	}
+	if len(aliases) > 1 {
+		return schema.NewAliasesGetAliasInternalServerError().WithPayload(&models.ErrorResponse{
```

Comment:
maybe we should return `4xx` here instead of `5xx`, internal server error suggests that a panic or an error occurred, which in this case didn't

Reply:
hmm, you're right, this is actually a server internal error, this situation should never happen